<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>senales cerebrales</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

font: 87 italic 8pt &quot;Arial Black&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="campo_grafico" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>341</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_cargar">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>390</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>cargar señal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_atras">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_adelante">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_aumentar">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_disminuir">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>tiempo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>amplitud</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="canal">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>390</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadCanal">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>390</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar Canal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccione numero de canal (1-8) ó * para mostrar todos los canales</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filtrar">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>330</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Filtrar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filtrado_tipo">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>300</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>280</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Digite 1 para Suave ó 0 para duro</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="campo_grafico2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>311</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Original</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Filtrada</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>420</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Primer Click -&gt; carga señal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>440</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Segundo Click -&gt; Muestra señal</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>785</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
